to_install.sh

```
@@ -0,0 +1 @@
+secator install tools cariddi==v1.4.1,fping==v5.3,gitleaks==v8.26.0,grype==v0.92.2,naabu==v2.3.4,nuclei==v3.4.4,subfinder==v2.7.1,trivy==v0.62.1
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)